recipes-devtools/mfgtools/mfgtoolslib_git.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "Freescale/NXP I.MX Chip image deploy tools library"
+SECTION = "devel"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=76b1b3cba07c1020780276c11101dd5a"
+
+DEPENDS = "libusb1 dtc"
+
+SRCREV = "7ad03a087644f703afeb3b5204c9a9192466627f"
+SRC_URI = " \
+    git://github.com/codeauroraforum/mfgtools;protocol=https \
+    file://0001-Fix-CXXFLAGS-in-CMakeList.patch \
+"
+
+PV = "1.0+git${SRCPV}"
+
+S = "${WORKDIR}/git"
+
+inherit cmake
+OECMAKE_SOURCEPATH = "${S}/MfgToolLib"
+
+do_install(){
+    install -d ${D}${libdir}
+    install -m 0644 ${B}/libMfgToolLibStatic.a ${D}${libdir}/libMfgToolLibStatic.a
+    install -d ${D}${includedir}/mfgtoolslib
+    install -m 0644 ${S}/MfgToolLib/*.h ${D}${includedir}/mfgtoolslib/
```

Comment:
The standard cmake install does not work? It'd be better to fix the CMakelists.txt

Also, why you don't use a single recipe to build the cli and lib?

Reply:
I fixed in a comment